org.lvnauth.LVNAuth.json

```diff
@@ -0,0 +1,110 @@
+{
+    "app-id": "org.lvnauth.LVNAuth",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "23.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "startup.sh",
+    "finish-args": [
+	    "--socket=x11",
+	    "--share=network",
+	    "--share=ipc",
+	    "--socket=pulseaudio",
+	    "--filesystem=home",
+	    "--device=dri"
+        ],
+    "modules": [
+        {
+            "name": "lvnauth",
+            "build-commands": [
+		"install -D flatpak/startup.sh ${FLATPAK_DEST}/bin/startup.sh",
+                "chmod +w ${FLATPAK_DEST}/bin/startup.sh",
+                "cp -r src/* ${FLATPAK_DEST}/bin",
+                "install -D flatpak/org.lvnauth.LVNAuth.desktop /app/share/applications/${FLATPAK_ID}.desktop",
+                "install -D flatpak/lvnauth_16x16.png /app/share/icons/hicolor/16x16/apps/${FLATPAK_ID}.png",
+                "install -D flatpak/lvnauth_32x32.png /app/share/icons/hicolor/32x32/apps/${FLATPAK_ID}.png",
+                "install -D flatpak/lvnauth_64x64.png /app/share/icons/hicolor/64x64/apps/${FLATPAK_ID}.png",
+                "install -D flatpak/lvnauth_128x128.png /app/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png",
+                "install -D flatpak/lvnauth_256x256.png /app/share/icons/hicolor/256x256/apps/${FLATPAK_ID}.png"
```

Comment:
I don't see any appstream file installed here. This is why the build did fail.

Reply:
When I add this to the json file:

`"install -D flatpak/org.lvnauth.LVNAuth.metainfo.xml /app/share/metainfo/${FLATPAK_ID}.metainfo.xml",`

I get an error when I attempt to test it using:
`flatpak run org.flatpak.Builder --install-deps-from=flathub --user --install --force-clean build-dir org.lvnauth.LVNAuth.json`

```
Errors were raised during this compose run:
general
  E: filters-but-no-output

org.lvnauth.LVNAuth
  E: metainfo-parsing-error
Refer to the generated issue report data for details on the individual problems.
Error: ERROR: appstreamcli compose failed: Child process exited with code 1

```

Reply:
does it validate?

still not installed. so the build won't pass.

Reply:
It validates now, I had a missing `</release>` section in the metainfo file and some invalid characters. I fixed them yesterday.
I used flatpak-builder-lint to check it.